main.py

```diff
@@ -2,6 +2,37 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for genre in genres:
+        Genre.objects.create(name=genre,)
+
+    for first_name, last_name in actors:
+        Actor.objects.create(
+            first_name=first_name,
+            last_name=last_name
+        )
+
+    Genre.objects.filter(name="Dramma",).update(name="Drama")
+    Actor.objects.filter(last_name="Klooney",).update(last_name="Clooney")
```

Comment:
There is no need to add `,` here and in the other places, remove it
```suggestion
    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
```